lib/pbench/test/unit/common/test_config.py

```diff
@@ -0,0 +1,68 @@
+"""Test PbenchConfig class and objects
+"""
+
+import pytest
+
+from pathlib import Path
+
+from pbench import PbenchConfig
+from pbench.common.exceptions import BadConfig
+
+
+_config_path_prefix = Path("lib/pbench/test/unit/common/config")
+
+
+class TestPbenchConfig:
+    def test_empty_config(self):
+        config = PbenchConfig(_config_path_prefix / "pbench.cfg")
+        assert config.TZ == "UTC", f"Unexpected TZ value, {config.TZ!r}"
+        assert (
+            config.log_fmt is None
+        ), f"Unexpected log format value, {config.log_fmt!r}"
```

Comment:
Question, whats `!r` does?

Reply:
It uses Python's `repr()` instead of `str()`.